### PR TITLE
Shared collection screen fixes

### PR DESCRIPTION
**Pull request by ashilkn**
Before :

before

After : 

https://user-images.githubusercontent.com/77285023/173063140-3afdca26-ec96-4699-872a-1eb2bd740c57.mov

After review : 

Screenshot from 2022-06-11 14-55-05
Screenshot from 2022-06-11 14-55-10

### CODE REVIEW

File: lib/ui/collections_gallery_widget.dart [hunk 1]
```diff
@@ -691,7 +690,9 @@ class SectionTitle extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
+      // margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
+      margin: EdgeInsets.fromLTRB(
+          (title == "Incoming" || title == "Outgoing") ? 16 : 24, 12, 0, 0),
```
Comment (vishnukvmd) [inline lib/ui/collections_gallery_widget.dart:695]: This check seems brittle. Perhaps let the wrapping container define the margins?

File: lib/ui/shared_collections_gallery.dart [hunk 2]
```diff
@@ -110,38 +111,48 @@ class _SharedCollectionGalleryState extends State<SharedCollectionGallery>
...
+                            collections.incoming[index]);
+                      },
+                      itemCount: collections.incoming.length,
+                      gridDelegate: SliverGridDelegateWithFixedCrossAxisCount(
+                          crossAxisCount: albumsCountInOneRow,
+                          mainAxisSpacing: 12,
+                          crossAxisSpacing: crossAxisSpacingOfGrid,
+                          childAspectRatio: sideOfThumbnail /
+                              (sideOfThumbnail +
+                                  24)), //24 is height of album title
```
Comment (vishnukvmd) [inline lib/ui/shared_collections_gallery.dart:146]: Adding a trailing `,` will make Dart remove unnecessary indendation

File: lib/ui/shared_collections_gallery.dart [hunk 3]
```diff
@@ -374,12 +385,19 @@ class IncomingCollectionItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    TextStyle albumTitleTextStyle = Theme.of(context)
+        .textTheme
+        .subtitle1
+        .copyWith(fontSize: 14, fontFamily: "Inter");
```
Comment (vishnukvmd) [inline lib/ui/shared_collections_gallery.dart:391]: Isn't the `fontFamily` set by default?

File: lib/ui/collections_gallery_widget.dart [hunk 4]
```diff
@@ -691,7 +690,9 @@ class SectionTitle extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Container(
-      margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
+      // margin: EdgeInsets.fromLTRB(24, 12, 0, 0),
```
Comment (vishnukvmd) [inline lib/ui/collections_gallery_widget.dart:693]: Remove comment?

File: lib/ui/collections_gallery_widget.dart (hunk 1 above)
Comment (ashilkn) [reply lib/ui/collections_gallery_widget.dart]: Have removed the check completely and and left padding is now 16 everytime.

**Review by vishnukvmd — APPROVED** [review]

**Pull request merged**